app-misc/prog-express/prog-express-3.7.8.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib
+
+DESCRIPTION="A modern and intuitive control software for the Batronix USB programming devices"
+HOMEPAGE="https://www.batronix.com"
+SRC_URI="amd64? ( http://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.amd64.deb )
+	x86? ( http://www.batronix.com/exe/Batronix/Prog-Express/deb/${P}-1.i386.deb )"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="prog-express"
+SLOT="0"
+
+RDEPEND="dev-libs/libusb
+	dev-db/sqlite
+	dev-dotnet/gtk-sharp
+	dev-dotnet/libgdiplus
+	dev-lang/mono
+	dev-lang/mono-basic
+	virtual/udev"
+DEPEND="app-arch/gzip
+	sys-devel/binutils"
+
+RESTRICT="strip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="usr/bin/bxusb usr/bin/bxusb-gui usr/bin/prog-express usr/sbin/bxfxload"
+
+src_unpack() {
+	# Unpack Debian package
+	if use amd64; then
+		ar x "${DISTDIR}"/${P}-1.amd64.deb || die
+	else
+		ar x "${DISTDIR}"/${P}-1.i386.deb || die
+	fi
+
+	# Unpack Debian package data file
+	unpack "${S}"/data.tar.xz
+}
+
+src_install() {
+	# Fix lib paths
+	for file in bxusb bxusb-gui prog-express; do
+		sed -i -e "s/lib/$(get_libdir)/g" "usr/bin/${file}" || die
+	done
+
+	# Install bin files
+	dobin "usr/bin/bxusb" "usr/bin/bxusb-gui" "usr/bin/prog-express"
+
+	# Install sbin files
+	dosbin "usr/sbin/bxfxload"
+
+	# Install lib files
+	insinto "/usr/$(get_libdir)"
+	doins -r "usr/lib/bxusb" "usr/lib/prog-express"
+
+	# Install usb config
+	insinto "/usr/$(get_libdir)/prog-express"
+	doins "${FILESDIR}"/pe.exe.config
+
+	# Install udev rule
+	insinto "/lib/udev/rules.d"
```

Comment:
please use `udev.eclass` for this

Reply:
Thanks, I will update the eBuild to use the udev class.